testool/src/main.rs

```diff
@@ -153,18 +160,34 @@ fn go() -> Result<()> {
         );
 
         // when running a report, the tests result of the containing cache file
-        // are used, but removing all Ignored tests
-        let mut results = if let Some(cache_filename) = args.cache {
-            let mut results = Results::from_file(PathBuf::from(cache_filename))?;
-            results
-                .tests
-                .retain(|_, test| test.level != ResultLevel::Ignored);
-            results
+        // are used, but by default removing all Ignored tests
+        // Another way is to skip the test which level not in whitelist_levels
+        let mut results_to_skip = if let Some(cache_filename) = args.cache {
```

Comment:
agree. previously `previous_results` a bit confusing because naming do not specify skipping purpose.  But I noticed code comments already explain clearly so just rollback the renaming!